### PR TITLE
Update Niv sources

### PR DESCRIPTION
**Pull request by expede**

### CODE REVIEW

File: shell.nix [hunk 1]
```diff
@@ -40,7 +40,6 @@ let
     ];
 
     haskell = [
-      unstable.haskell-language-server
```
Comment (walkah) [inline shell.nix:43]: `LSP :: The following servers support current file but do not have automatic installation configuration: lsp-haskell` 

but was it working for you?

Comment (expede) [reply shell.nix:43]: Weird. Yeah, I have `lsp-haskell` via the `doom-emacs` module.

Comment (expede) [reply shell.nix:43]: From the `lsp-haskell` [docs](https://github.com/emacs-lsp/lsp-haskell#emacs-configuration):

> lsp-mode automatically requires the lsp-haskell package , so you do not need to require lsp-haskell unless you like being explicit.

Comment (expede) [reply shell.nix:43]: 🤦‍♀️ ZOMG no, I have it in my `home-manager`

Comment (expede) [reply shell.nix:43]: Also, landing soon:

> haskell-language-server: 0.9.0 -> 1.0.0 

https://github.com/NixOS/nixpkgs/commit/3c81fd41eec6e285c1e0e3a17fa2df39d74e911c

**Comment by expede** [comment]
Step 1: Put HLS back in as a dependency
Step 2: Watch it compile forever
Step 3: Profit?

I really do wonder if it's just easier to manually grab from the releases section for now:

Screen Shot 2021-03-12 at 4 10 03 PM

**Comment by expede** [comment]
> /home/runner/.stack/setup-exe-cache/x86_64-linux-nix/Cabal-simple_mPHDZzAJ_3.2.1.0_ghc-8.10.4: startProcess: runInteractiveProcess: exec: does not exist (No such file or directory)

<Internal screaming intensifies>

**Pull request closed without merging**